bundler/bin/parallel_rspec

```diff
@@ -4,6 +4,8 @@
 require_relative "../spec/support/rubygems_ext"
 
 require_relative "../spec/support/switch_rubygems"
+
+require "ostruct" # Backfill missing require in turbo_tests. Remove when added upstream
```

Comment:
This is fixed in `turbo_tests` v2.2.1 (ref: https://github.com/serpapi/turbo_tests/pull/49#issuecomment-2046576834).

I'm sorry about this issue.

Reply:
No problem at all! Thanks for the quick release!